src/pages/organize/[orgId]/people/lists/[viewId]/shared.tsx

```diff
@@ -57,7 +67,7 @@ export const getServerSideProps: GetServerSideProps = scaffold(async (ctx) => {
 
   // TODO: Handle this some other way with server-side models?
   const apiClient = new BackendApiClient(ctx.req.headers);
-  const accessLevel = await getAccessLevel(
+  let accessLevel = await getAccessLevel(
```

Comment:
I think, for a first step, we should begin with keeping this as a const, and then add a line below to do something like `if (accessLevel == null) { return notFound() }`. This would make it 404 for the user. Would you wanna try this approach?